packages/clay-css/src/images/icons/minus-circle.svg

```diff
@@ -4,6 +4,6 @@
 *
 * SPDX-License-Identifier: BSD-3-Clause
 -->
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-	<path class="lexicon-icon-outline" d="M256.4,0c-340.8,0-342.8,512,0,512C595.2,512,599.2,0,256.4,0z M384.7,304H128.1c-64.2,0-63.2-96,0-96h256.6C448.8,208,450.8,304,384.7,304z"></path>
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+	<path class="lexicon-icon-outline" fill-rule="evenodd" clip-rule="evenodd" d="M8 .002a8 8 0 1 1 0 16 8 8 0 0 1 0-16Zm-3.5 6.5a1.5 1.5 0 0 0 0 3h7a1.5 1.5 0 0 0 0-3h-7Z"/>
```

Comment:
@SelenaAungst It was mostly good. It was missing `class="lexicon-icon-outline"` here. `yarn compile` depends on this class to add `fill="#{$color}"` in `_lx-icons-generated.scss`. I realized we have a lot of icons without `fill` in that file.